Multisig/UI/Safe Management/Add Safe/SafeLoadedViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  SafeLoadedViewController.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 05.10.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class SafeLoadedViewController: AccountActionCompletedViewController {
+
+    private var safe: Safe!
+
+    convenience init() {
+        self.init(namedClass: AccountActionCompletedViewController.self)
+    }
+
+    override func viewDidLoad() {
+        do {
+            safe = try Safe.getSelected()!
+            descriptionText = "\((safe.name ?? "Safe")) is read-only. Would you like to add owner key for this Safe to confirm transactions?"
+            accountName = safe.name
+            accountAddress = safe.addressValue
+        } catch {
+            fatalError()
+        }
+        titleText = "Load Gnosis Safe"
+        headerText = "Safe loaded"
+        primaryActionName = "Add owner key"
+        secondaryActionName = "Skip"
+
+        super.viewDidLoad()
+    }
+
+    override func primaryAction(_ sender: Any) {
+        Tracker.trackEvent(.userOnboardingOwnerAdd)
```

Comment:
for tracking I would define "trackingEvent" in base class and here we just assign the value of it

Reply:
I'll handle it as part of a separate issue for tracking, leaving as is for now